scripts/clean

```diff
@@ -0,0 +1,14 @@
+#!/bin/sh -e
+
+if [ -d 'dist' ] ; then
+    rm -r dist
+fi
+if [ -d 'site' ] ; then
+    rm -r site
+fi
+if [ -d 'htmlcov' ] ; then
+    rm -r htmlcov
+fi
+if [ -d 'uvicorn.egg-info' ] ; then
+    rm -r uvicorn.egg-info
+fi
```

Comment:
In my experience this script has been made obsolete since switching to the "publish via GH actions" workflow. I don't think we absolutely need it. WDYT?

Reply:
yup !  in the trash